modules/persistent-storage-csi-smb-cifs-provision-static.adoc

```diff
@@ -0,0 +1,216 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-provision-static_{context}"]
+= Static provisioning
+
+You can use static provisioning to create a persistent volume (PV) and persistent volume claim (PVC) to consume existing Server Message Block protocol (SMB) shares:
+
+.Prerequisites
+* Access to the {product-title} web console.
+* {FeatureName} CSI Driver Operator and driver installed.
+* You have installed the SMB server and know the following information about the server:
+** Hostname
+** Share name
+** Username and password
+
+.Procedure
+
+To set up static provisioning:
+
+. Create a Secret for access to the Samba server using the following command with the following example YAML file:
++
+[source,terminal]
+--
+$ oc create -f <file_name>.yaml
+--
++
+[source,yaml]
+.Secret example YAML file
+--
+apiVersion: v1
+kind: Secret
+metadata:
+  name: smbcreds <1>
+  namespace: samba-server <2>
+stringData:
+  username: <username> <3>
+  password: <password> <4>
+--
+<1> Name of the Secret for the Samba server.
+<2> Namespace for the Secret for the Samba server.
+<3> Username for the Secret for the Samba server.
+<4> Password for the Secret for the Samba server.
+
+. Create a PV by running the following command with the following example YAML file:
++
+[source,terminal]
+----
+$ oc create -f <pv_file_name>.yaml <1>
+----
+<1> The name of the PV YAML file.
++
+.Example PV YAML file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  annotations:
+    pv.kubernetes.io/provisioned-by: smb.csi.k8s.io
+  name: <pv_name> <1>
+spec:
+  capacity:
+    storage: 100Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: ""
```

Comment:
@mpatlasov Can you please make sure this actually works?

cc @Phaow

Reply:
Hi @gcharot @mpatlasov , I could confirm that if we use the `storageClassName: ""` in the `PersistentVolume` then we also need set the same storageclass name in the pvc definition. Otherwise `persistentvolume-controller` will complain `Cannot bind to requested volume "manual-pv": storageClassName does not match`.

Reply:
thanks @Phaow good call!

Reply:
Seems to work for me too. PVC goes to Bound state.